documentation/src/main/wiki/PerformanceTestsSummary.md

```diff
@@ -0,0 +1,13 @@
+# Performance Tests Summary
+
+## Introduction
+This page gathers all tests summaries, recommendations and actions undertaken.
+
+## Overview
+* Throughput of Knot.x varies from **223 requests/sec** up to **840 requests/second** for different scenarios for **1 second response time**.
+* Knot.x is stable.
+* Knot.x recovers after crashes.
+* Several tweaks and fixes introduced to [[Knot.x, Mocks and Apache HTTP Server|PerformanceTestsTuning]]
```

Comment:
`* Several fixes applied to Knot.x as well as configuration tweaks both for Knot.x and Apache HTTP Server`
Just to state clearly we didn't fixed Apache HTTP but we fixed configuration of it.